### PR TITLE
Fix duplicate setting of keyword parameter, test more versions, add badges

### PR DESCRIPTION
**Pull request by ScottPJones**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,12 @@ os:
   - osx
 julia:
   - 0.5
-  # - nightly
+  - 0.6
+  - nightly
+matrix:
+  allow_failures:
+    - julia: 0.6
```
Comment (sglyon) [inline .travis.yml]: While we are at it, would you mind removing 0.6 from here -- I'd like to ensure that those tests are passing.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
-[![Join the chat at https://gitter.im/spencerlyon2/PlotlyJS.jl](https://badges.gitter.im/spencerlyon2/PlotlyJS.jl.svg)](https://gitter.im/spencerlyon2/PlotlyJS.jl?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Join the chat at https://gitter.im/sglyon/PlotlyJS.jl](https://badges.gitter.im/sglyon/PlotlyJS.jl.svg)](https://gitter.im/sglyon/PlotlyJS.jl?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-[![Build Status](https://travis-ci.org/spencerlyon2/PlotlyJS.jl.svg?branch=master)](https://travis-ci.org/spencerlyon2/PlotlyJS.jl)
+[![Build Status](https://travis-ci.org/sglyon/PlotlyJS.jl.svg?branch=master)](https://travis-ci.org/sglyon/PlotlyJS.jl)
+
+#[![AppVeyor](https://ci.appveyor.com/api/projects/status/mue0n1yhlxq4ok8d/branch/master?svg=true)](https://ci.appveyor.com/project/sglyon/plotlyjs-jl/branch/master)
+#[![Coverage Status](https://img.shields.io/coveralls/sglyon/PlotlyJS.jl.svg)](https://coveralls.io/r/sglyon/PlotlyJS.jl)
+
+[![PlotlyJS](http://pkg.julialang.org/badges/PlotlyJS_0.4.svg)](http://pkg.julialang.org/?pkg=PlotlyJS)
```
Comment (sglyon) [inline README.md]: We don't test on 0.4 anymore, so we shouldn't show this badge, right?

File: appveyor.yml [hunk 3]
```diff
@@ -2,8 +2,18 @@ environment:
   matrix:
   - JULIAVERSION: "julialang/bin/winnt/x86/0.4/julia-0.4-latest-win32.exe"
   - JULIAVERSION: "julialang/bin/winnt/x64/0.4/julia-0.4-latest-win64.exe"
```
Comment (sglyon) [inline appveyor.yml]: Let's drop 0.4 testing on App app veyor like we did on travis and a few lines down don't allow 0.6 tests to fail.

File: .travis.yml (hunk 1 above)
Comment (ScottPJones) [inline .travis.yml]: OK, sure!

File: README.md (hunk 2 above)
Comment (ScottPJones) [inline README.md]: Yes!  (I'm returning from Europe this weekend, so may be delayed by a day)

Comment (ScottPJones) [inline README.md]: Never mind! Got a moment to do it!

**Comment by sglyon** [comment]
Thanks!

**Pull request merged**